### PR TITLE
Group datasource

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves https://github.com/Twingate/terraform-provider-twingate/issues/100

## Changes
- added group datasource

### CODE REVIEW

File: twingate/datasource_group.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			"id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The ID of the Group",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The name of the Group",
+			},
```
Comment (vmanilo) [inline twingate/datasource_group.go:45]: @alexmensch please check descriptions for group datasource

**Review by ekampf — APPROVED** [review]

**Review by ekampf — APPROVED** [review]

File: docs/data-sources/group.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  id = "<your group's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the Group
```
Comment (alexmensch) [inline docs/data-sources/group.md:26]: Change to:

`The ID of the Group. The ID for the Group must be obtained from the Admin API.`

**Review by alexmensch — APPROVED** [review]
Pre-approving, please ask if you have any questions on the documentation updates before merging

File: docs/data-sources/group.md (hunk 2 above)
Comment (vmanilo) [reply docs/data-sources/group.md]: updated description

**Pull request merged**